http/write_handler.go

```diff
@@ -160,19 +160,31 @@ func (h *WriteHandler) handleWrite(w http.ResponseWriter, r *http.Request) {
 	points, err := models.ParsePointsWithPrecision(data, time.Now(), req.Precision)
 	if err != nil {
 		logger.Info("Error parsing points", zap.Error(err))
-		EncodeError(ctx, err, w)
+		EncodeError(ctx, &platform.Error{
+			Code: platform.EInvalid,
+			Op:   "http/handleWrite",
+			Err:  err,
+		}, w)
 		return
 	}
 
 	exploded, err := tsdb.ExplodePoints(org.ID, bucket.ID, points)
 	if err != nil {
 		logger.Info("Error exploding points", zap.Error(err))
-		EncodeError(ctx, err, w)
+		EncodeError(ctx, &platform.Error{
+			Code: platform.EInvalid,
+			Op:   "http/handleWrite",
```

Comment:
This one should be an internal server error rather than EInvalid

```go
Msg: fmt.Printf("unable to convert points to internal structures: %v", err),
```